src/cf-forms/usage.md

```diff
@@ -213,57 +187,26 @@ See the 'Form icons' section below for guidance on adding icons to states.
 </div>
 ```
 
-#### Error icon
-
-- The icon must be placed directly after the form input in the markup and the input must use the 'error' class.
-- For invalid fields only use the alert role to call attention to fields that need immediate attention: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role
-
-<input class="error" type="text" value="Invalid input" title="Test input">
-<span class="cf-form_input-icon cf-icon cf-icon-delete-round" role="alert"></span>
-
-```
-<input class="error" type="text" value="Invalid input" title="Test input">
-<span class="cf-form_input-icon cf-icon cf-icon-delete-round" role="alert"></span>
-```
-
-#### Warning icon
-
-<input class="warning" type="text" value="Invalid input" title="Test input">
-<span class="cf-form_input-icon cf-icon cf-icon-error-round" role="alert"></span>
-
-```
-<input class="warning" type="text" value="Invalid input" title="Test input">
-<span class="cf-form_input-icon cf-icon cf-icon-error-round" role="alert"></span>
-```
-
-#### Success icon
-
-<input class="success" type="text" value="Validated input" title="Test input">
-<span class="cf-form_input-icon cf-icon cf-icon-approved-round"></span>
-
-```
-<input class="success" type="text" value="Validated input" title="Test input">
-<span class="cf-form_input-icon cf-icon cf-icon-approved-round"></span>
-```
-
-
 ## Buttons
 
 ### Input and button
 
 <div class="o-form__input-w-btn">
     <div class="o-form__input-w-btn_input-container">
-        <input type="text" title="Test input">
+        <input class="a-text-input" type="text" title="Test input">
     </div>
     <div class="o-form__input-w-btn_btn-container">
         <button class="a-btn">Search</button>
     </div>
 </div>
 
+##NOTE: Fix the button text size
```

Comment:
Ah, glad you found it cause I forgot already. It's showing up as 14px and I need to investigate why.